lib/inline_svg/webpack_asset_finder.rb

```diff
@@ -9,11 +11,29 @@ def initialize(filename)
     end
 
     def pathname
-      public_path = Webpacker.config.public_path
       file_path = Webpacker.instance.manifest.lookup(@filename)
-      return unless public_path && file_path
+      return unless file_path
+
+      if Webpacker.dev_server.running?
+        asset = open("#{Webpacker.dev_server.protocol}://#{Webpacker.dev_server.host_with_port}#{file_path}")
```

Comment:
I'm a little nervous about using `open-uri` here but given this is for `Webpacker.dev_server` mode only _and_ we're constructing the URL manually it should be fine.

Reply:
For what it's worth, in the custom `AssetFinder` class we have running in our app, we do this to not use `Kernel#open`:

```ruby
asset = Net::HTTP.get(Webpacker.dev_server.host, file_path, Webpacker.dev_server.port)

Tempfile.new(file_path).tap do |file|
  file.binmode
  file.write(asset)
  file.rewind
end
```

It's still pretty readable, and doesn't have the same potential security concerns!

Reply:
Thanks for the suggestion, @agrobbin. 

@moxie how do you feel about using `Net::HTTP` instead of `Kernel.open` for reading the asset? 🤔

Reply:
@jamesmartin perfectly fine with me. Will update asap.

Reply:
`URI.open` is also a valid option, or at least better than `Kernel.open`